src/corehost/cli/deps_resolver.cpp

```diff
@@ -692,11 +704,16 @@ bool deps_resolver_t::resolve_probe_dirs(
         }
     }
     
-    for (const auto& entry : fx_entries)
+    // Add fx package locations to fx_dir
+    for (int i = 1; i < m_fx_definitions.size(); ++i)
     {
-        if (!add_package_cache_entry(entry, m_fx_dir))
+        const auto& fx_entries = m_portable ? m_fx_definitions[i]->get_deps().get_entries(asset_type) : empty;
```

Comment:
How could we ever get in here if `m_portable` is false?

Reply:
Removed the check. I believe the code used to start with `0` instead of `1` thus the check